pkg/cmd/get/clusters.go

```diff
@@ -21,14 +67,296 @@ func preClustersE(cmd *cobra.Command, args []string) error {
 }
 
 func list(cmd *cobra.Command, args []string) error {
-	provider := cluster.NewProvider(cluster.ProviderWithDocker())
+	clusters, err := populateClusterList()
+	if err != nil {
+		return err
+	} else {
+		// Convert the list of the clusters to Table of clusters
+		printTable(printers.PrintOptions{}, generateClusterTable(clusters))
+		return nil
+	}
+}
+
+func populateClusterList() ([]Cluster, error) {
+	logger := helpers.CmdLogger
+
+	detectOpt, err := util.DetectKindNodeProvider()
+	if err != nil {
+		//logger.Error(err, "failed to detect the provider.")
```

Comment:
remove commented out logger message like these

Reply:
Removed the comments. See commit: 9fc54e2

Reply:
See: 9fc54e2